src/sql/workbench/services/query/common/queryRunner.ts

```diff
@@ -276,6 +280,17 @@ export default class QueryRunner extends Disposable {
 		this._onMessage.fire([message]);
 	}
 
+	/**
+	 * Used to update the Server Connection ID of the current connection as assigned by the server after query complete.
+	 */
+	public handleServerConnId(id: string): void {
+		let idEvent: QueryConnID = {
```

Comment:
nit: It's confusing to refer to it both as "Server connection id" and "query connection id" so I'd stick with one and use it consistently everywhere (with query connection id being my slight preference)